mr/src/test/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandlerBaseTest.java

```diff
@@ -512,4 +770,14 @@ protected void createHiveTable(String tableName, String location) {
   protected String locationForCreateTable(String tempDirName, String tableName) {
     return null;
   }
+
+  private List<Record> createTableWithGeneratedRecords(Schema schema, int numRecords, long seed, String tableName)
+          throws IOException {
+    List<Record> records = TestHelper.generateRandomRecords(schema, numRecords, seed);
+    createTable(tableName, schema, records);
+    // sanity check, fetch all rows
+    List<Object[]> allRows = shell.executeStatement("SELECT * from default." + tableName);
+    Assert.assertEquals("Number of rows doesn't match expected.", records.size(), allRows.size());
```

Comment:
Do we need this test? This adds an extra sql statement which could be costly when run 24 times. I think that if the number of rows are not correct when preparing the table, then the tests will fail anyway.
Or did I miss something?

Reply:
You are right, removed the unnecessary query/assert,

Reply:
@pvary, do you think that we should move some of these Hive tests to an integration test suite? Then we could update the CI action to run Hive integration tests separately and hopefully avoid triggering it for changes to Spark or Flink.

Reply:
@rdblue: Sounds like a good idea. Do we have anything like this already set-up for any of the subprojects, or we should invent/find the path?
Thanks, Peter

Reply:
@pvary, Jack created integration tests for the Glue catalog, so you can look at what he did in iceberg-glue. Thanks!